app/models/invenio_rdm_record_converter.rb

```diff
@@ -130,7 +130,7 @@ def invenio_metadata(gf)
       "description": gf.description.first,
       "additional_descriptions": additional_descriptions(gf.description),
       "publisher": gf.publisher.shift,
-      "publication_date": "#{gf.date_uploaded.year}-#{gf.date_uploaded.month}-#{gf.date_uploaded.day}",
+      "publication_date": format_publication_date(gf.date_uploaded),
```

Comment:
For the `publication_date`, I thought we wanted to use `gf.date_created` first (or something like it) and then fall back on `gf.date_uploaded`. For instance https://digitalhub.northwestern.edu/files/7bcdc672-2525-4a80-b0a1-143de6cddbc6 was uploaded in our time but created in 1893-03-07 . In the following box correspondence document https://northwestern.app.box.com/file/684887467855?s=1e1ogwnz4vrh7upmrlfuf3nk4vw1ap64 publication_date in InvenioRDM is associated with date created in Digitalhub.

Honestly, I still have some issues figuring out where these box files are, which ones are the right ones and whether or not we encountered additional information that changed things... That box link was right last time I checked with Sara though in December 2020.